lib/schema/schema.go

```diff
@@ -321,6 +321,12 @@ const manifestSchema = `
             "generic": {"$ref": "#/definitions/providerGeneric"}
           }
         },
+        "ingress": {
+          "type": "object",
+          "properties": {
```

Comment:
Maybe have a subsection specific to nginx as well, similar to openebs below, in case we add other controllers in the future?

```yaml
ingress:
  nginx:
    enabled: true
```